CHANGELOG.md

```diff
@@ -2,10 +2,39 @@
 
 ## unreleased
 
-* [FEATURE] OTLP receiver: Add new option `otlp.promote_resource_attributes`, for any OTel resource attributes that should be promoted to metric labels. #14200
-* [ENHANCEMENT] OTLP receiver: Warn when encountering exponential histograms with zero count and non-zero sum. #14706
+## 2.55.0-rc.0 / 2024-09-17
+
+* [CHANGE] TSDB: Only count unknown record types in `record_decode_failures_total` metric. #14042
```

Comment:
I could be swayed that this doesn't need to be announced as a CHANGE.

Reply:
:+1:  it was originally proposed as a bug fix too: https://github.com/prometheus/prometheus/pull/14042/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR6

Reply:
I got it now.  That PR is both a bugfix and a misfeature.  I'll downgrade it from CHANGE.